### PR TITLE
hammer: set node restrictions in FindStorageNodesRequest

### PR DESCRIPTION
**Pull request by kaloyan-raev**

### CODE REVIEW

File: satellite/metainfo/metainfo.go [hunk 1]
```diff
@@ -129,11 +129,21 @@ func (endpoint *Endpoint) CreateSegment(ctx context.Context, req *pb.SegmentWrit
...
+	}
+
+	maxPieceSize := eestream.CalcPieceSize(req.GetMaxEncryptedSegmentSize(), redundancy)
+
 	// TODO most probably needs more params
 	request := &pb.FindStorageNodesRequest{
 		Opts: &pb.OverlayOptions{
-			Amount:       int64(req.Redundancy.Total),
-			Restrictions: &pb.NodeRestrictions{},
+			Amount: int64(req.Redundancy.Total),
```
Comment (mniewrzal) [inline satellite/metainfo/metainfo.go]: Do you think we still need this TODO above?

Comment (kaloyan-raev) [reply satellite/metainfo/metainfo.go]: No idea. It was not me who put it :)
If you want, I can remove it.

Comment (mniewrzal) [reply satellite/metainfo/metainfo.go]: I mean, do you think we have all the necessary params? :)

Comment (kaloyan-raev) [reply satellite/metainfo/metainfo.go]: I think they are enough for now. I will remove the TODO comment.

**Pull request merged**